package.json

```diff
@@ -228,6 +228,12 @@
         "title": "View Schema",
         "category": "Confluent: Schema"
       },
+      {
```

Comment:
New action bound to a schema subject group to be able to view the most recent version w/o having to expand the tree container for the subject group.